spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/support/ResettableServletInputStreamWrapper.java

```diff
@@ -0,0 +1,38 @@
+package org.springframework.cloud.netflix.zuul.filters.support;
```

Comment:
Sorry one more thing, we need a license header here

Reply:
my bad. done :)